### PR TITLE
Update the ImageSeriesReader documentation.

### PR DESCRIPTION
**Pull request by zivy**
Updated the documentation to cover the reading of meta-data
dictionaries.

Change-Id: Ie53e2820609ef7308ed72e75e4086ce6885d5b15

### CODE REVIEW

File: Code/IO/include/sitkImageSeriesReader.h [hunk 1]
```diff
@@ -27,7 +27,18 @@ namespace itk {
   namespace simple {
 
     /** \class ImageSeriesReader
-     * \brief Read series of image into a SimpleITK image
+     * \brief Read series of image into a SimpleITK image.
```
Comment (blowekamp) [inline Code/IO/include/sitkImageSeriesReader.h]: Please change the singular "image" to the plural  "image files" for clarity.

File: Code/IO/include/sitkImageSeriesReader.h [hunk 2]
```diff
@@ -162,14 +173,11 @@ namespace itk {
...
-   *     programmatically add a meta-data dictionary to the compounded image by reading in
-   *     one or more images from the series using the ImageFileReader
-   *     class, analyzing the meta-dictionary associated with each of
-   *     those images and creating one that is relevant for the
-   *     compounded image.
+   *     image will have an empty meta-data dictionary.
+   *     If you need the meta-data dictionaries associated with each
+   *     slice then you should use the ImageSeriesReader class.
    *
-   * \sa itk::simple::ImageFileReader for reading a single file
```
Comment (blowekamp) [inline Code/IO/include/sitkImageSeriesReader.h]: I think this See Also should be kept.

**Comment by blowekamp** [comment]
Addresses issue #240 

Thank you for the prompt fix! I added a couple minor comments.

File: Code/IO/include/sitkImageSeriesReader.h (hunk 2 above)
Comment (zivy) [inline Code/IO/include/sitkImageSeriesReader.h]: Done.

**Pull request merged**